### PR TITLE
Update build and task configurations

### PR DESCRIPTION
**Pull request by andschwa**
Small mostly developer oriented fixes.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -459,7 +459,7 @@
               "type": "PowerShell",
               "request": "launch",
               "script": "^\"\\${file}\"",
-              "cwd": "^\"\\${file}\""
+              "cwd": "^\"\\${workspaceFolder}\""
```
Comment (andschwa) [inline package.json:462]: @rkeithhill Do you know why this was `file` before? It seemed quite wrong to me, as the thing is asking for a directory, and our other task (below) was given `workspaceFolder` too.

File: vscode-powershell.build.ps1 [hunk 2]
```diff
@@ -106,7 +100,7 @@ task UpdateReadme -If { $script:IsPreviewExtension } {
     }
 }
 
-task Package UpdateReadme, {
+task Package UpdateReadme, Build, {
```
Comment (andschwa) [inline vscode-powershell.build.ps1:103]: Annoying dependency bug if you've cleaned and then want to package, it should just build.

File: .vscode/launch.json [hunk 3]
```diff
@@ -47,6 +47,12 @@
         "${workspaceFolder}/out/**/*.js"
       ],
       "preLaunchTask": "Build"
-    }
+    },
+    {
+      "name": "PowerShell Launch Current File",
```
Comment (andschwa) [inline .vscode/launch.json:52]: I'm using this _a lot_ when testing, high time to put it in the project's config.

**Review by rjmholt — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (andschwa) [reply package.json:462]: I found my answer at long last...https://github.com/PowerShell/vscode-powershell/blob/3dc11c1fd671bd814dce75bffdff5f737aa817fe/src/features/DebugSession.ts#L304-L306

**Pull request merged**